README.md

```diff
@@ -6,4 +6,32 @@
 
 This repository contains an implementation of the StyleCop rules using the .NET Compiler Platform. Where possible, code fixes are also provided to simplify the process of correcting violations.
 
+## Using StyleCop.Analyzers
+
+The preferable way to use the analyzers is to add the nuget package [StyleCop.Analyzers](http://www.nuget.org/packages/StyleCop.Analyzers/)
```

Comment:
:bulb:

StyleCopAnalyzers can be installed using the NuGet Package Manager in Visual Studio 2015 RC.

![image](https://cloud.githubusercontent.com/assets/1408396/8233513/491f301a-159c-11e5-8b7a-1e16a0695da6.png)